### ISSUE TITLE
Startup fails when running from systemctl

### ISSUE THREAD

**Issue by HT154** [realm/realm-mobile-platform#56]
Ubuntu 16.04.1

Running via `systemctl start realm-object-server.service` causes it to fail. Here's the result from `systemctl status realm-object-server.service`. Nothing is appended to the log file (which I verified is writeable).
```
● realm-object-server.service - Realm Sync Services
   Loaded: loaded (/etc/systemd/system/realm-object-server.service; enabled; vendor preset: enabled)
   Active: inactive (dead) (Result: exit-code) since Thu 2016-11-24 23:02:01 UTC; 6s ago
  Process: 16074 ExecStart=/usr/bin/realm-object-server -c /etc/realm/configuration.yml (code=exited, status
 Main PID: 16074 (code=exited, status=1/FAILURE)

Nov 24 23:02:01 ht154 systemd[1]: realm-object-server.service: Main process exited, code=exited, status=1/FA
Nov 24 23:02:01 ht154 systemd[1]: realm-object-server.service: Unit entered failed state.
Nov 24 23:02:01 ht154 systemd[1]: realm-object-server.service: Failed with result 'exit-code'.
Nov 24 23:02:01 ht154 systemd[1]: realm-object-server.service: Service hold-off time over, scheduling restar
Nov 24 23:02:01 ht154 systemd[1]: Stopped Realm Sync Services.
Nov 24 23:02:01 ht154 systemd[1]: realm-object-server.service: Start request repeated too quickly.
Nov 24 23:02:01 ht154 systemd[1]: Failed to start Realm Sync Services.
```

Running it with the exact same command that systemctl is using (`/usr/bin/realm-object-server -c /etc/realm/configuration.yml`) works fine and starts the server as expected. The log file is used as expected.

The issue started happening after updating Realm Object Server via apt. I can't remember which version it started happening with, but I'm currently running 1.0.0-BETA-4.2

**Comment by teotwaki** [comment]
Hi @HT154,

Any chance you could share the log file with us?

Thanks

**Comment by HT154** [comment]
Nothing gets logged when starting via systemctl.

Here is the log file (level: all) when starting directly from the shell: https://gist.github.com/HT154/d9e09fb361df227a568a05bb871beab4

**Comment by HT154** [comment]
I ended up backing up my config, running `apt-get purge`, and reinstalling the package. It's working now, but I'm still not sure what the actual issue was.

**Comment by teotwaki** [comment]
@HT154,

Any chance you could share the configuration file (new and old/broken) with us? You can email help@realm.io if you'd rather not disclose it publicly.

Thanks,

**Comment by HT154** [comment]
Not able to get the config right now, but it was pretty standard. I think the only modifications from the default were HTTPS-related.

**Comment by teotwaki** [comment]
Thanks @HT154, we'll look into that.

**Comment by insidegui** [comment]
Any news? I'm also having this issue. Reinstalling didn't fix it for me.

[Here's the configuration file](https://github.com/realm/realm-mobile-platform/files/1110655/configuration.yaml.zip)

The only setting I changed was the root path where I specified a block storage that is correctly formatted and mounted (I'm running on DigitalOcean).

If I start the server by running the command manually with the same config it works, using `systemctl` it doesn't.

[This is the log file](https://github.com/realm/realm-mobile-platform/files/1110658/realm.log.zip)
